feat(rust): tuify space delete (no args) command

Lumexralph opened the pull request:
<!-- Thank you for sending a pull request :heart: -->

## Current behavior

As described in this issue #6460

<!-- Please describe the current behavior of the code before the changes in this pull request are applied. -->

## Proposed changes

The `ockam space delete` is made to be interactive as suggested in this issue #6460
<!-- Please describe the changes proposed in this pull request. -->
<!-- If this pull request resolves an already recorded bug or a feature request, please add a link to that issue. -->

## Checks

<!--
To help us review and merge this pull request quickly, please confirm the following
by replacing the [ ] in front of each bullet point below with [x]
-->

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

<!-- We're looking forward to merging your contribution!! -->

Lumexralph committed:
feat(rust): tuify space delete (no args) command

adrianbenavides commented on implementations/rust/ockam/ockam_command/src/space/delete.rs:
```diff
@@ -36,37 +37,97 @@ pub struct DeleteCommand {
+    }
+
+    fn cmd_arg_confirm_deletion(&self) -> bool {
+        false
```
I think we should add this feature to this command. It would be as simple as adding the `yes` argument as it's done in [other commands](https://github.com/build-trust/ockam/blob/54bb211c0f1eb20af324cc5e37b38f3a43ff9817/implementations/rust/ockam/ockam_command/src/relay/delete.rs#L31-L33).

Lumexralph replied:
Makes sense!

adrianbenavides reviewed (commented):
Hey @Lumexralph thanks for the PR! I left a comment with a small improvement we could make 👍

Lumexralph committed:
feat(space-delete): pass the command to tui struct

Instead of hardcoding the confirm_deletion,
the DeleteCommand was passed to the DeleteTui
 to access the "yes" attribute.

adrianbenavides reviewed (approved):
Looks perfect now, thanks a lot @Lumexralph for your contribution! 🙏 

Your PR will get merged through https://github.com/build-trust/ockam/pull/6969
